CODEOWNERS

```diff
@@ -142,7 +142,7 @@
 /samples/dataprotect/                       @mozilla-mobile/act
 /samples/firefox-accounts/                  @grigoryk
 /samples/glean/                             @mozilla-mobile/telemetry
-/samples/sync-logins/                       @ekager
+/samples/sync-logins/                       @grigoryk
```

Comment:
@grigoryk I added you to `samples-sync-logins` since your name was on `service-sync-logins`, but I can change this to something else (default ACT team) if that's better. 🙂